docs/streams/upgrade-guide.html

```diff
@@ -34,13 +34,7 @@ <h1>Upgrade Guide and API Changes</h1>
     </div>
 
     <p>
-        If you want to upgrade from 1.1.x to 1.2.0 and you have customized window store implementations on the <code>ReadOnlyWindowStore</code> interface
-        you'd need to update your code to incorporate the newly added public APIs; otherwise you don't need to make any code changes.
-        See <a href="#streams_api_changes_120">below</a> for a complete list of 1.2.0 API and semantic changes that allow you to advance your application and/or simplify your code base.
-    </p>
-
-    <p>
-        If you want to upgrade from 1.0.x to 1.1.0 and you have customized window store implementations on the <code>ReadOnlyWindowStore</code> interface
+        If you want to upgrade from 1.0.x to 1.1.1 and you have customized window store implementations on the <code>ReadOnlyWindowStore</code> interface
```

Comment:
Hmm.. seems this cherry-picking not correct, as it is for `trunk` the upgrade section should include for `1.2.0` right?